llvm-bazel/llvm-project-overlay/llvm/include/llvm/Config/config.h

```diff
@@ -0,0 +1,364 @@
+
+/*===------- llvm/Config/llvm-config.h - llvm configuration -------*- C -*-===*/
```

Comment:
```suggestion
/*===------- llvm/Config/llvm-config.h - llvm configuration -------*- C -*-===*/
```

(At least, I think? Or does the .cmake file actually have a blank line?)

Reply:
Nope it does not. It also doesn't have a license header (and probably this license header should say the right filename :-D)